python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -109,204 +97,195 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
+    ? [
+        {
+          label: "Resources",
+          value:
+            Object.entries(actor.usedResources).length > 0 &&
+            Object.entries(actor.usedResources)
+              .sort((a, b) => a[0].localeCompare(b[0]))
+              .map(
+                ([key, value]) =>
+                  `${sum(
+                    value.resourceSlots.map((slot) => slot.allocation),
+                  )} ${key}`,
+              )
+              .join(", "),
+        },
+        {
+          label: "Number of pending tasks",
+          value: actor.taskQueueLength.toLocaleString(),
+          tooltip:
+            "The number of tasks that are currently pending to execute on this actor. If this number " +
+            "remains consistently high, it may indicate that this actor is a bottleneck in your application.",
+        },
+        {
+          label: "Number of executed tasks",
+          value: actor.numExecutedTasks.toLocaleString(),
+          tooltip:
+            "The number of tasks this actor has executed throughout its lifetimes.",
+        },
+        {
+          label: "Number of ObjectRefs in scope",
+          value: actor.numObjectRefsInScope.toLocaleString(),
+          tooltip:
+            "The number of ObjectRefs that this actor is keeping in scope via its internal state. " +
+            "This does not imply that the objects are in active use or colocated on the node with the actor " +
+            `currently. This can be useful for debugging memory leaks. See the docs at ${memoryDebuggingDocLink} ` +
+            "for more information.",
+        },
+        {
+          label: "Number of local objects",
+          value: actor.numLocalObjects.toLocaleString(),
+          tooltip:
+            "The number of small objects that this actor has stored in its local in-process memory store. This can be useful for " +
+            `debugging memory leaks. See the docs at ${memoryDebuggingDocLink} for more information`,
+        },
+        {
+          label: "Object store memory used (MiB)",
+          value: actor.usedObjectStoreMemory.toLocaleString(),
+          tooltip:
+            "The total amount of memory that this actor is occupying in the Ray object store. " +
+            "If this number is increasing without bounds, you might have a memory leak. See " +
+            `the docs at: ${memoryDebuggingDocLink} for more information.`,
+        },
+      ]
+    : [
+        {
+          label: "Actor ID",
+          value: actor.actorId,
+          tooltip: "",
+        },
+        {
+          label: "Required resources",
+          value:
+            actor.requiredResources &&
+            Object.entries(actor.requiredResources).length > 0 &&
+            Object.entries(actor.requiredResources)
+              .sort((a, b) => a[0].localeCompare(b[0]))
+              .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+              .join(", "),
+          tooltip: "",
```

Comment:
Can you update tooltips to explain this field in details? Like the creation cannot be performed because resources are not avialable in raylet or something.